src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -350,12 +351,25 @@ const basketItemsQuery = gql`query basketItemsQuery(
 				}
 			}
 			totals {
+				bonusAppliedTotal
+				bonusAvailableTotal
 				creditAmountNeeded
-				creditAvailableTotal
 				creditAppliedTotal
+				creditAvailableTotal
 				donationTotal
 				itemTotal
+				freeTrialAppliedTotal
+				freeTrialAvailableTotal
+				kivaCardTotal
+				kivaCreditAvailableTotal
+				kivaCreditAppliedTotal
+				kivaCreditRemaining
+				kivaCreditToReapply
 				loanReservationTotal
+				redemptionCodeAppliedTotal
+				redemptionCodeAvailableTotal
+				universalCodeAppliedTotal
+				universalCodeAvailableTotal
```

Comment:
I'll be doing cleanup passes later to move this entire inline query out into it's own file and use the shopTotals fragment.